pkg/datapath/config.go

```diff
@@ -120,3 +122,15 @@ type ConfigWriter interface {
 	// of configurable options for the endpoint to the specified writer.
 	WriteEndpointConfig(w io.Writer, cfg EndpointConfiguration) error
 }
+
+// GetRemoteSNATDstAddrExclusionCIDR returns a CIDR for SNAT exclusion. Any
+// packet sent from a local endpoint to an IP address belonging to the CIDR
+// should not be SNAT'd.
+func GetRemoteSNATDstAddrExclusionCIDR() *cidr.CIDR {
```

Comment:
nit: No need to `Get..()`; AFAIK usually we avoid this in Go unless there's ambiguity with a public field of the same name.

```suggestion
func RemoteSNATDstAddrExclusionCIDR() *cidr.CIDR {
```